tests/Bridge/OpenAI/DallETest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpLlm\LlmChain\Tests\Bridge\OpenAI;
+
+use PhpLlm\LlmChain\Bridge\OpenAI\DallE;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\Attributes\Small;
+use PHPUnit\Framework\Attributes\Test;
+use PHPUnit\Framework\TestCase;
+
+#[CoversClass(DallE::class)]
+#[Small]
+class DallETest extends TestCase
```

Comment:
```suggestion
final class DallETest extends TestCase
```

Reply:
I will change the finals because it seem to be a rule but just out of curiosity ... what is the profit of having the tests being final? If it is a style thingy it should be enforced by the style tooling? Otherwise it is looking like a "stupid work" thingy during a review that one want to not do for themself regulary. What do you think?

Reply:
One thing I like is, that it will not popup when you extend a class as it is not possible to extend them. besides that, if I read `final` I know, I don't need to think about another consumer or not. If no `final` there could be some class which extends this class

Reply:
Thanks! So it is "just" a style thing. Like me adding the `final` keyword on classes implementing an interface (that could be mocked / stubbed in tests instead). 

Then we should think about having the cs-fixer or rector doing it to get rid of such monkey comments. Do you know if there would be something available? My experience with php-cs-fixer is not that much.